test/functional/treesitter/highlight_spec.lua

```diff
@@ -838,3 +838,67 @@ describe('treesitter highlighting (help)', function()
   end)
 
 end)
+
+describe('treesitter highlighting (nested injections)', function()
```

Comment:
Since this is a regression it's usually good to include the issue number.

Reply:
I put it in the `it()` because I was thinking the `describe()` group could be more general, but should it be both?

Reply:
Ah sorry I missed that. (Reviewing on mobile)